docs/proposals/scheduling-suspension/README.md

```diff
@@ -0,0 +1,208 @@
+该字段为空，表示默认允许调度，从而不影响现有行为。
+-->
+
+By expanding the `ResourceBinding.ResourceBindingSpec.Spec.Suspension` field, and add the `Suspension.Scheduling` field to its structure to indicate suspension of scheduling.
+The Karmada Scheduler component needs to decide whether to start scheduling the corresponding `ResourceBinding/ClusterResourceBinding` at this time based on the value of `Suspension.Scheduling`.
+When `Suspension.Scheduling` is true, scheduling is suspended and the corresponding conditions are refreshed. When `Suspension.Scheduling` is false, scheduling is resumed. `ResourceBinding` created by default
+This field is empty, indicating that scheduling is allowed by default and does not affect existing behavior.
+
+### API Change
+
+```go
+type ResourceBindingSpec struct {
+    // Suspension declares the policy for suspending different aspects of propagation.
+    // nil means no suspension. no default values. 
+    // +optional 
+    Suspension *policyv1alpha1.Suspension `json:"suspension,omitempty"`
+}
+
+// Suspension defines the policy for suspending different aspects of propagation.
+type Suspension struct {
+    // Dispatching controls whether dispatching should be suspended.
+    // nil means not suspend, no default value, only accepts 'true'.
+    // Note: true means stop propagating to all clusters. Can not co-exist
+    // with DispatchingOnClusters which is used to suspend particular clusters.
+    // +optional
+    Dispatching *bool `json:"dispatching,omitempty"`
+
+    // DispatchingOnClusters declares a list of clusters to which the dispatching
+    // should be suspended.
+    // Note: Can not co-exist with Dispatching which is used to suspend all.
+    // +optional
+    DispatchingOnClusters *SuspendClusters `json:"dispatchingOnClusters,omitempty"`
+	
+    // New Added
+    // Scheduling controls whether scheduling should be suspended.
+    // nil means not suspend, no default value, only accepts 'true'.
+    // Karmada scheduler will pause scheduling when value is true and resume scheduling when it's nil.
+    Scheduling *bool `json:"scheduling,omitempty"`
+}
+```
+
+### Implementations
+
+<!--
+Karmada scheduler监听rb状态，当ResourceBinding.Suspension.Scheduling=true时忽略该RB资源，暂停调度，当ResourceBinding.Suspension.Scheduling=false时恢复调度。
+并且当RB已经调度成功后，不再允许修改rb从非暂停到暂停，同时ResourceBinding.Suspension.Scheduling设置为true或者false时，在RB中的status中刷新相应的condition。
+-->
+Karmada scheduler listens the `ResourceBinding` resource , ignores the `ResourceBinding` resource when `ResourceBinding.Suspension.Scheduling`=true, pauses scheduling, and resumes scheduling when `ResourceBinding.Suspension.Scheduling`=false.
+
+And when the `ResourceBinding` has been successfully scheduled, it is no longer allowed to modify the `ResourceBinding` from non-suspended to suspended, and when `ResourceBinding.Suspension.Scheduling` is set to true or false, the corresponding condition is refreshed in the status of the `ResourceBinding`.
```

Comment:
> And when the `ResourceBinding` has been successfully scheduled, it is no longer allowed to modify the `ResourceBinding` from non-suspended to suspended,

It sounds like we need to introduce a validating webhook to force this, right? Can you explain why we need to do that?

Reply:
Yeah I have considered that, becaue the scheduling suspension happened before scheduling success, and after rb is successfully scheduled it's no necessary to suspend rb again, which will also confuse users.

Reply:
I think the `work` suspension has the same problem.

Reply:
> and after rb is successfully scheduled it's no necessary to suspend rb again, which will also confuse users.

Yeah, I agree, it brings ambitious if we don't force that.
I will track this on #5977.